README.md

```diff
@@ -173,23 +197,42 @@ To learn more about development with MAGE and Docker, visit the
   [here](https://memgraph.com/download). We offer Ubuntu, Debian and CentOS
   based Memgraph packages. To install, proceed to the following
   [site](https://memgraph.com/docs/memgraph/installation).
-- To build and install MAGE query modules you will need: **Python3**, **Make**,
-  **CMake**, **Clang**, **UUID** and **Rust**.
+- To build and install MAGE query modules you will need:
+  - libcurl4
+  - libpython3.9
+  - libssl-dev
+  - openssl 
+  - build-essential
+  - make     
+  - cmake          
+  - curl           
+  - g++          
+  - python3      
+  - python3-pip 
+  - python3-setuptools    
+  - python3-dev     
+  - clang  
+
+Since Memgraph needs to load MAGE's modules, there is the `setup` script to help you. With `setup` script you can build modules, so Memgraph
```

Comment:
Grammar
```suggestion
Since Memgraph needs to load MAGE's modules, there is the `setup` script to help you. With it, you can build the modules so that Memgraph
```